packages/quick_blue/example/lib/main.dart

```diff
@@ -53,11 +53,27 @@ class _MyAppState extends State<MyApp> {
         ),
         body: Column(
           children: [
+            // StreamBuilder<AvailabilityState>(  // Uncomment on iOS and macOS
+            //   stream: QuickBlue.availabilityChangeStream,
+            //   builder: (context, snapshot) {
+            //     String availability;
+            //     switch (snapshot.data) {
```

Comment:
```
extension AvailabilityStateExt on AvailabilityState {
  String toString() {
    switch() {
      case xx: return 'xx';
    }
  }
}
```

Reply:
I put it in the model and overrode toString() since this cannot be done in an extension.